Gemfile

```diff
@@ -24,6 +24,7 @@ group :development, :test do
   gem "debug", platforms: %i[mri windows], require: "debug/prelude"
   gem "i18n-tasks"
   gem "l10nizer", require: false
+  gem "pry"
```

Comment:
If we want to add pry, it should be in its own PR, rather than sneaking in alongside some other work, especially when we already have [ruby-debug] in the project (which provides the same functionality and is probably more featured and mature at this point).

[ruby-debug]: https://github.com/ruby/debug